src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+| 1. Has a withdrawal address been provided?   | Proceed to #2                         | **No** withdrawal will be processed, validator skipped |
+| 2. Is the validator still active?            | Proceed to #3                         | **Full** withdrawal will be processed                  |
+| 3. Is the effective balance maxed out at 32? | **Rewards payment** will be processed | **No** withdrawal will be processed, validator skipped |
+
+A maximum of 16 withdrawals can be processed in a single block. At that rate, 115,200 validator withdrawals can be processed per day (assuming no missed blocks). As noted above, validators without eligible withdrawals will be skipped, decreasing the time to finish the sweep.
+
+## Frequently asked questions {#faq}
+
+<ExpandableCard title="Once I have provided a withdrawal address, can I change it to an alternative withdrawal address?">
+No, the process to provide withdrawal credentials is a one-time process, and cannot be changed once submitted.
+</ExpandableCard>
+
+<ExpandableCard title="What if I participate in liquid staking derivatives or pooled staking">
+<p>If you are part of a staking pool or hold liquid staking derivatives, you should check with your provider for more details about how staking withdrawals will affect your arrangement, as each service operates differently.</p>
+<p>In general, users will likely have nothing they need to do, and these services will no longer be limited by the inability to withdrawal rewards or exit validator funds after this upgrade.</p>
+<p>This means that users can now decide to redeem their underlying staked ETH, or change which staking provider they utilize. If a particular pool is getting too large, funds can be exited and redeemed, and re-staked with a <a href="https://pools.invis.cloud">smaller provider</a>. Or, if you’ve accumulated enough ETH you could <a href="/staking/solo/">stake from home</a>.</p>
+</ExpandableCard>
+
+<ExpandableCard title="Do rewards payments (partial withdrawals) happen automatically?">
+<p>Yes, as long as your validator has provided a withdrawal address. This must be provided once to enable any withdrawals, then rewards payments will be automatically triggered every few days with each validator sweep.</p>
+</ExpandableCard>
+
+<ExpandableCard title="Do full withdrawals happen automatically?">
+<p>No, if your validator is still active on the network, a full withdrawal will not happen automatically. This requires manually initiating a voluntary exit.</p>
+<p>Once a validator has completed the exiting process, and assuming the account has withdrawal credentials, the remaining balance will <em>then</em> be withdrawal during the next validator sweep.</p>
+</ExpandableCard>
+
+<ExpandableCard title="How can I withdrawal a custom amount?">
+<p>Withdrawals are designed to be pushed automatically, transferring any ETH that is not actively contributing to stake.</p>
```

Comment:
I'd add for clarity something like "any amount beyond 32 ETH"

Reply:
This sentence was written this way specifically to sum up the criteria for ANY withdrawal..
- If you're active, you can have up to 32 ETH at stake. The portion "not actively contributing to stake" is only the ETH above 32.
- But, if you've exited, the portion "not actively contributing to stake" is 100% of it.

I do understand this may not be intuitive though, so perhaps we break it up and be more explicit about the two types.

Reply:
Left a note elsewhere too about some nuances with "any amount beyond 32 ETH", since this also requires that your _effective balance_ is 32. 

Explainer: It is possible to have a total balance over 32 (but less than 32.25), and have your effective balance be 31... this could happen if you're offline and drop to 31.75 total balance, which would cause your effective balance to drop to 31. To get back to 32 effective, you actually need to get your total balance to 32.25 first. This is due to what are called hysteresis zones where an effective balance will not change, and are explained in [this Attestant post]](https://www.attestant.io/posts/understanding-validator-effective-balance/)

This was also intentionally kept concise as an FAQ answer since there is more explanation above about what you're mentioning... Just wanted to make it clear you can't request custom amounts since it's all automatically processed. Feel free to push back though, not against expanding this a little if people prefer.